test/e2e/suite_test.go

```diff
@@ -50,10 +50,48 @@ var (
 		medusaRestoreDone       pollingConfig
 		datacenterUpdating      pollingConfig
 	}
+)
 
-	logKustomizeOutput = flag.Bool("logKustomizeOutput", false, "")
-	logKubectlOutput   = flag.Bool("logKubectlOutput", false, "")
-	imageTag           = flag.String("imageTag", "latest", "")
+var (
+	configFileFlag = flag.String(
+		"configFile",
```

Comment:
What do you think about renaming this to make it clear it is a kubeconfig file?

Reply:
Indeed, I will change this to `kubeconfigFile`.